subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -1082,6 +1037,33 @@ static void done_disabled_cb(void *param)
 	flush(param);
 }
 
+static void flush_safe(void *param)
+{
+	struct ll_scan_aux_set *aux;
+	struct ull_hdr *hdr;
+	uint8_t ref;
+
+	/* Flush from here or from done event, if one is
+	 * pending.
+	 */
```

Comment:
```suggestion
	/* ref == 0
	 * All PDUs were scheduled from LLL and there is no pending done
	 * event, we can flush here.
	 *
	 * ref == 1
	 * There is pending done event so we need to flush from disabled
	 * callback. Flushing here would release aux context and thus
	 * ull_hdr before done event was processed.
	 */
```
In my humble opinion this version of comment, taken from changed code is more informative, what do you think?

Reply:
Done